.goreleaser.yml

```diff
@@ -1,18 +1,22 @@
 # This is an example .goreleaser.yml file with some sane defaults.
 # Make sure to check the documentation at http://goreleaser.com
+
+env:
+  - CGO_ENABLED=0
+
 before:
   hooks:
     # You may remove this if you don't use go modules.
     - go mod download
     # you may remove this if you don't need go generate
     - go generate ./...
 builds:
-  - env:
-      - CGO_ENABLED=0
-    goos:
+  - goos:
       - linux
       - windows
       - darwin
+    binary: ./vault/plugins/vault-plugin-secrets-kubernetes-reader
```

Comment:
This seems different than what's defined in Dockerfile.goreleaser. Am I missing something?

Reply:
They copy the binary to the temp dir from the appropriate dist directory. Please see https://github.com/revit13/vault-plugin-secrets-kubernetes-reader/runs/4751567755?check_suite_focus=true#step:5:483. I ran it with debug flag.